ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4596,3 +4596,262 @@ AHU_DFSS_DX2ZC_ECONZ_CO2C_DSP_RCKTM2X_HTZC_CC2XM_RTM_DTM:
   - DTM
   opt_uses:
   - condensing_fan_current_sensor
+  
+AHU_CHWVM:
```

Comment:
not valid: what kind of AHU is it? even if these are the only fields you get for the type, you still need to model it as the AHU type it really is (controls to zone temp, has dx or chw cooling, outside air monitoring, etc.). The translation will be adjusted to handle the missing fields. we do not want a bunch of bespoke AHUs which are really incomplete versions of a real, canonical AHU.

Reply:
Deleted the single Canonicals and have created unified canonials.